lib/hooks/userspace/hook-grpc.js

```diff
@@ -36,13 +93,15 @@ module.exports = function(version_, agent_) {
     return {};
   }
   return {
-    'src/node/src/grpc_extension.js': {
-      patch: function(extension) {
+    'src/node/src/client.js': {
+      patch: function(client) {
         agent = agent_;
-        shimmer.wrap(extension.Call.prototype, 'startBatch', startBatchWrap);
+        shimmer.wrap(client, 'makeClientConstructor',
+            makeClientConstructorWrap);
       },
-      unpatch: function(extension) {
-        shimmer.unwrap(extension.Call.prototype, 'startBatch');
+      unpatch: function(client) {
+        shimmer.unwrap(client, 'makeClientConstructor');
+        agent_.logger.info('gRPC makeClientConstructor: unpatched');
```

Comment:
If I understand the semantics correct, any Client objects that have been already constructed may continue to be traced (i.e. will remain unaffected) by the unpatch.